spec/features/admin/admin_sees_order_details_spec.rb

```diff
@@ -72,11 +53,13 @@
     expect(page).to have_content "Item subtotal: $50.65"
 
     expect(page).to have_content "Order total: $50.65"
-    expect(page).to have_content "Order status: paid"
+    expect(page).to have_content "Order status: ordered"
   end
 
-  xscenario 'can view item page through order details' do
-    click_on "Order created on 06/09/15"
+  scenario 'can view item page through order details' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.